cogboard-webapp/src/components/widgets/types/JiraBucketWidget.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { array } from 'prop-types';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import TableCell from '@material-ui/core/TableCell';
+import TableBody from '@material-ui/core/TableBody';
+
+const JiraBucketWidget = ({ bucketNames, issueCounts }) => {
+  const buckets = bucketNames.map((bucket, index) => (
```

Comment:
update to a model that doesn't have this ordering dependency

Reply:
Changed